packages/cli/templates/default-app/package.json.hbs

```diff
@@ -20,5 +22,8 @@
     "cross-env": "^7.0.0",
     "lerna": "^3.20.2",
     "prettier": "^1.19.1"
+  },
+  "resolutions": {
+    "@backstage/cli": "file:/home/runner/work/backstage/backstage/packages/cli"
```

Comment:
Guess this is WIP but it should only be present when running through the test script. Should probably also link out to all public packages, or we'll end up with a bit of desync. I imagine looking through packages/** and plugins/** and adding entries for every package that isn't private should work for the foreseeable future.

Reply:
Yeah, very much wip. Working on it now 👍